env.example

```diff
@@ -36,6 +36,9 @@ export DEPLOY_VERSION=v3.11
 # into OpenShift
 #export AZURE_AAD_GROUP_ADMINS_ID=9e4f44bd-90a1-44b2-b89a-ed9884662958
 
+# Azure App insights key for monitoring shim to send data
+# export AZURE_APP_INSIGHTS_KEY=
```

Comment:
this insights key... should we add it to secrets?

Reply:
Already added